exonum/src/runtime/dispatcher/schema.rs

```diff
@@ -40,6 +43,19 @@ const INSTANCES: &str = "dispatcher_instances";
 const PENDING_INSTANCES: &str = "dispatcher_pending_instances";
 const LOCAL_MIGRATION_RESULTS: &str = "dispatcher_local_migration_results";
 const INSTANCE_IDS: &str = "dispatcher_instance_ids";
+const SERVICE_INTERFACES: &str = "dispatcher_interfaces";
+
+#[derive(Debug, Serialize, Deserialize, BinaryValue)]
+#[binary_value(codec = "bincode")]
```

Comment:
IMO, the use of `bincode` is unwarranted here. It can be gotten rid of by storing services in a `Group` of `KeySetIndex`es keyed by the instance ID, for example.